command.py

```diff
@@ -1,14 +1,18 @@
-import discord
+"""
+Implementing CalicoBot commands
+"""
 import datetime
-from discord.ext import commands
-from discord import app_commands
 import time
 import embeds
-import main
+import discord
+from discord.ext import commands
+from discord import app_commands
 
-bot=commands.Bot(command_prefix="t?",intents=discord.Intents.all())
+user_karma_list=list()
+def command_init(main_list):
```

Comment:
Very sorry,I forgot this description.
```suggestion
def command_init(main_list):
    """A function to initialize command module

    Args:
        main_list (list): input your user karma data
    """
```